not sure if related, but: 

```
======================================================================
FAIL: test_case_close_format (corehq.apps.repeaters.tests.test_repeater.CaseRepeaterTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 232, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 205, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 220, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 201, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/repeaters/tests/test_repeater.py", line 275, in test_case_close_format
    CaseFactory().close_case(CASE_ID)
  File "corehq/ex-submodules/casexml/apps/case/mock.py", line 320, in close_case
    return self.create_or_update_case(CaseStructure(case_id=case_id, attrs={'close': True}))[0]
  File "corehq/ex-submodules/casexml/apps/case/mock.py", line 323, in create_or_update_case
    return self.create_or_update_cases([case_structure], form_extras)
  File "corehq/ex-submodules/casexml/apps/case/mock.py", line 346, in create_or_update_cases
    return list(CaseAccessors(self.domain).get_cases(case_ids, ordered=True))
  File "/mnt/commcare-hq-ro/corehq/form_processor/interfaces/dbaccessors.py", line 274, in get_cases
    return self.db_accessor.get_cases(case_ids, ordered=ordered)
  File "submodules/dimagi-utils-src/dimagi/utils/decorators/memoized.py", line 11, in _inner
    return m(*args, **kwargs)
  File "submodules/dimagi-utils-src/dimagi/utils/decorators/memoized.py", line 128, in __call__
    value = self.func( *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/form_processor/interfaces/dbaccessors.py", line 265, in db_accessor
    if should_use_sql_backend(self.domain):
  File "/mnt/commcare-hq-ro/corehq/form_processor/utils/general.py", line 24, in should_use_sql_backend
    domain_name, domain_object = _get_domain_name_and_object(domain_object_or_name)
  File "/mnt/commcare-hq-ro/corehq/form_processor/utils/general.py", line 51, in _get_domain_name_and_object
    return domain_object_or_name, Domain.get_by_name(domain_object_or_name)
  File "/mnt/commcare-hq-ro/corehq/util/quickcache.py", line 312, in inner
    return helper(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/quickcache.py", line 201, in __call__
    return super(SkippableQuickCache, self).__call__(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/util/quickcache.py", line 100, in __call__
    content = self.fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/domain/models.py", line 537, in get_by_name
    _assert(False, '%r is not a valid domain name' % name)
  File "/mnt/commcare-hq-ro/corehq/util/soft_assert/core.py", line 33, in __call__
    return self._call(assertion, msg, obj)
  File "/mnt/commcare-hq-ro/corehq/util/soft_assert/core.py", line 47, in _call
    raise AssertionError(msg)
AssertionError: None is not a valid domain name
```

--- a/corehq/form_processor/utils/general.py
+++ b/corehq/form_processor/utils/general.py
@@ -45,7 +45,9 @@ def _should_use_sql_backend_in_tests(domain_object):
 
 def _get_domain_name_and_object(domain_object_or_name):
     from corehq.apps.domain.models import Domain
-    if isinstance(domain_object_or_name, Domain):
+    if domain_object_or_name is None:
+        return None, None
+    elif isinstance(domain_object_or_name, Domain):
         return domain_object_or_name.name, domain_object_or_name
     elif getattr(settings, 'DB_ENABLED', True):
         return domain_object_or_name, Domain.get_by_name(domain_object_or_name)

